robot_smach_states/src/robot_smach_states/human_interaction/answer_questions.py

```diff
@@ -144,15 +68,15 @@ def answer_crowd_questions(action):
         return True
 
 
-def answer_random_gender(action):
+def answer_random_gender(_, __):
     return random.choice(["A male", "A female"])
 
 
-def answer_predefined_questions(action):
+def answer_predefined_questions(action, _):
     return action['solution']
 
 
-def answer_count_people(action, crowd_data):
+def answer_count_people(action, _, crowd_data):
```

Comment:
`action` argument unused, replace by `_` (and thus second argument by `__`)

Reply:
Disagree, `action`  is used in an if in line 98